### ISSUE TITLE
Allow to manually browse or indicate a SD folder instead of an hardcoded one.

### ISSUE THREAD

**Issue by PoorPocketsMcNewHold** [sunryze-git/SDMU#1]
![image](https://github.com/sunryze-git/SDMU/assets/13170204/0c59fe95-7bdd-4b7a-8b54-304ecf9d57a5)
It may be because you haven't thought to test it or expected people to test it on other platforms such as Linux.

**Comment by sunryze-git** [comment]
Should be pretty easy to implement. Linux support was not extensively tested so you are right on that. My primary systems are a Windows PC and a Mac.

**Comment by sunryze-git** [comment]
I have pushed some fixes to main. Not sure if they are working, but it involves correctly formatting on each OS, and correctly identifying if we have superuser permissions or not.

I'll test it when I can get access to my Linux system at home.

**Comment by PoorPocketsMcNewHold** [comment]
![image](https://github.com/sunryze-git/SDMU/assets/13170204/de8f49b3-71ec-4be6-a88f-7ebdd97e53cc)
Sadly, it doesn't seem to make much of a difference.

**Comment by sunryze-git** [comment]
Yeah, there still is more fixes, I locally have gotten something, though it requires your device to already be mounted under /mnt or /media. 

I can confirm it does work, and i'll push the commit to main today.
